doc/reference.md

```diff
@@ -330,17 +330,24 @@ constexpr string_view enum_type_name() noexcept;
 
 ```cpp
 template <typename... Es>
-[[nodiscard]] constexpr optional<std::uintmax_t> enum_fuse(Es... values);
+[[nodiscard]] constexpr std::optional<FusedEnum> enum_fuse(Es... values);
```

Comment:
optional not necessary means std::optional.

Reply:
fixed.